whist/core/cards/card.py

```diff
@@ -1,11 +1,13 @@
 """Card related classes"""
 
 from enum import Enum
-from typing import Any, Optional
+from functools import total_ordering
+from typing import Any, Optional, Iterator
 
 from pydantic import BaseModel
 
 
+@total_ordering
```

Comment:
Is this replacing the __ge__ and others?

Reply:
yes. i only have to implement __eq__ and one comparison and then everything else is added automatically